pyproject.toml

```diff
@@ -35,6 +35,7 @@ types-pillow = "^9.5.0.4"
 mypy-protobuf = "^3.4.0"
 tox = "^4.5.1"
 isort = "^5.12.0"
+pytest-watcher = "^0.2.6"
```

Comment:
[q] is this needed?

Reply:
Not strictly, but it was helpful to have this utility when I was spiking out unit tests. I can you remove if you prefer.

Reply:
Went ahead and removed